pkg/workceptor/kubernetes.go

```diff
@@ -55,15 +56,16 @@ type kubeUnit struct {
 
 // kubeExtraData is the content of the ExtraData JSON field for a Kubernetes worker
 type kubeExtraData struct {
-	Image         string
-	Command       string
-	Params        string
-	KubeNamespace string
-	KubeVerifyTLS bool
-	KubeTLSCAData string
-	KubeConfig    string
-	KubePod       string
-	PodName       string
+	Image             string
+	Command           string
+	Params            string
+	KubeNamespace     string
+	KubeVerifyTLS     bool
+	KubeTLSCAData     string
+	KubeConfig        string
+	KubePod           string
+	PodName           string
+	PodPendingTimeout string
```

Comment:
I think I'd rather have this on the kubeUnit struct than kubeExtraData

Reply:
I have made the required changes.